.bandit

```diff
@@ -0,0 +1,9 @@
+[bandit]
```

Comment:
![image](https://github.com/ijcode17/getting-started/assets/35231827/a1afb029-d9bc-44ef-9df7-71a8494999ab)

Remove this files